hw/ip/otp_ctrl/data/otp_ctrl_img_creator_sw_cfg.hjson

```diff
@@ -0,0 +1,84 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+//
+// Use the gen-otp-img.py script to convert this configuration into
+// a MEM file for preloading the OTP in FPGA synthesis or simulation.
+//
+
+{
+    // The partition and item names must correspond with the OTP memory map.
+    partitions: [
+        {
+            name:  "CREATOR_SW_CFG",
+            items: [
+                {
+                    name:  "CREATOR_SW_CFG_DIGEST",
+                    value: "0x0",
+                },
+                {
+                    name: "CREATOR_SW_CFG_USE_SW_RSA_VERIFY",
+                    // Use software mod_exp implementation for signature
+                    // verification. See the definition of `hardened_bool_t` in
+                    // sw/device/lib/base/hardened.h.
+                    value: "0x739",
```

Comment:
I think we should leave it as is since we are planning to switch to the OTBN implementation only after verifying that it works reliably. Thus, TEST_UNLOCKED, RMA, and probably DEV should use the SW implementation by default. WDYT?